dataactbroker/handlers/agency_handler.py

```diff
@@ -43,9 +45,15 @@ def get_cgacs_without_sub_tier_agencies(sess=None):
     return sess.query(CGAC).filter(CGAC.cgac_id.notin_([st.cgac.cgac_id for st in cgac_sub_tiers])).all()
 
 
-def get_accessible_agencies():
+def get_accessible_agencies(perm_level='reader', perm_type='mixed'):
     """ List all CGAC and FREC Agencies user has DABS permissions for
 
+        Args:
+            perm_level: only return agencies the user has 'reader', 'writer', or 'submitter' affiliations
+                        (Note: the higher the level, the more gets filtered out. Thus, the reader level includes all
+                        available agencies)
+            perm_type: only return agencies with either 'dabs', 'fabs', or 'mixed' affiliations
```

Comment:
this doesn't really encompass what this does. More like `filters results to agencies the user has 'dabs' or 'fabs' affiliations for. 'mixed' results in no filtering` or something like that